process/ReleaseProcess.md

```diff
@@ -0,0 +1,159 @@
+# Release Process and Guidelines
+
+## What is a Zowe release
+
+A release of the Zowe project consists of two 'convenience' deliverables - CLI binaries, and a PAX file which contains the Zowe API Mediation Layer, Zowe Desktop, and Zowe Explorer-Server. The CLI binaries are obtainable either through standard npm install commands, or, for users with limited internet access, through a convience zip file which allows for offline installation. The CLI convenience zip and Zowe PAX are both available through the official [Zowe Website](https://www.zowe.org).
```

Comment:
Technically - these aren't Zowe open source project deliverables, but those done under the previous beta program between CA, Rocket and IBM. Right now the only official deliverables would be source code.

Reply:
We are continuing to create the convenience builds.  What would you recommend here?  Do you want to strike this or reword?  I would agree that the only "official" release is the source but we do deliver a large pax with installers for all the individual Zowe projects.